tests/Core/Content/Product/SalesChannel/Listing/ProductRouteBase.php

```diff
@@ -197,17 +199,31 @@ public function testShopwareHandlesRequestWhenFindologicisInactive(): void
         $this->call($productRoute, $request, $salesChannelContextMock);
     }
 
+    public function testCustomCriteriaIsAllowed(): void
+    {
+        $salesChannelContextMock = $this->getMockedSalesChannelContext(true);
+        $request = Request::create('http://your-shop.de/some-category');
+        $request->setSession($this->getSessionMock());
+
+        $criteria = new Criteria();
+        $criteria->addFilter(new EqualsFilter('product.name', 'yeet'));
+
+        $productRoute = $this->getRoute();
+        $this->call($productRoute, $request, $salesChannelContextMock, '69', $criteria);
```

Comment:
This test seem to not assert anything or check any method calls, can this be improved?

Reply:
Well not really, as before this would've resulted in an exception.

Reply:
So there is no way to verify `yeet` is used?

Reply:
Sure, I used the response to ensure the criteria contains the filter.